OpenRA.Mods.Common/Scripting/Global/PaletteGlobal.cs

```diff
@@ -0,0 +1,36 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+using OpenRA.Mods.Common.Traits;
+using OpenRA.Scripting;
+
+namespace OpenRA.Mods.Common.Scripting
+{
+	[ScriptGlobal("Palette")]
```

Comment:
Day/night cycles were included in my "etc." :)

I'm just seeing this from a scripter's point of view. Many of those don't know and don't care that you're achieving the effect with modifying a palette, so referring to that in the API makes it less accessible IMO. The API should instead reflect what the scripter actually wants to do, not how it's done behind the scenes.